langchain-core/src/utils/image.ts

```diff
@@ -0,0 +1,12 @@
+import * as fs from "node:fs/promises";
```

Comment:
We definitely want this to work in web envs too

Reply:
I believe this still should work because we're conditionally importing this only if you pass in a file path.

Correct me if I'm wrong about how the imports work though